language/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -323,6 +325,24 @@ fn script(
     }))
 }
 
+/// Generate a mapping from numerical address and module name to named address, for module's whose
+/// identities contained a named address.
+fn address_names<'a>(
+    dependencies: impl Iterator<Item = &'a ModuleIdent>,
+) -> HashMap<(MoveAddress, &'a str), Symbol> {
```

Comment:
That's right, just querying here -- in general, I use `HashMap` when order shouldn't matter, and `BTreeMap` otherwise.  I feel like we could write a clippy rule or something for iterating over HashMap and HashSet?  I'll look into it.